tests/integration_tests/bugs/test_gh586.py

```diff
@@ -0,0 +1,42 @@
+"""Integration test for pull #586
+
+If a non-default AuthorizedKeysFile is specified in /etc/ssh/sshd_config,
+ensure we can still ssh as expected.
+"""
+import paramiko
+import pytest
+from io import StringIO
+from tests.integration_tests.assets import get_test_rsa_keypair
+
+
+public_rsa_key, private_rsa_key = get_test_rsa_keypair()
```

Comment:
Capitalise these?

Reply:
Capitalize RSA or treat the variables as constants? If just RSA, I don't think it's typical to capitalize abbreviations in normal variables names. If treat like constants, the contents are derived from reading a file, so I'd say their immutable, not constant, and thus should remain lowercase.

Reply:
I'm happy to leave it as-is. I've never really considered all caps to mean constant, so much as it means "module-level" but I guess I'll take you agreeing with PEP-8 as a victory. :wink:

Reply:
*ahem*
![image](https://user-images.githubusercontent.com/153674/101094923-c611ad80-3582-11eb-9a7f-1d919745af60.png)

Reply:
> can reach agreement on this issue

This sort of passive aggressive behaviour is very self-defeating. :grin:

Reply:
Haha, all in good fun (at least for me :wink: )!

Reply:
Absolutely!